### ISSUE TITLE
Allow users to exclude the desired build 

### ISSUE THREAD

**Issue by ehsandeep** [anshumanbh/brutesubs#8]
sometime due to time consuming or ineffective result by some of the build/tool like enumall ro any of, so
it will be good if users will get option to exclude the tool he/she don't want to use it , eg : **docker-compose up -enumall** or something like that.

**Comment by anshumanbh** [comment]
@s4ndeeps 

From what I understand, you're basically asking to run individual tools but that sort of defeats the purpose of this orchestration all together. I have gobuster, enumall and sublist3r running simultaneously because I feel they bruteforce most of the domains, if not all. And, then altdns after that. 

Right now, the orchestration kind of relies on gobuster to run first, merge the wordlists provided and then run gobuster, sublist3r and enumall simultaneously. So, If I ran either sublist3r or enumall separately, that merge functionality would be missing. And, if I include that in every script, that kind of breaks the orchestration. So, its a catch 22 there. 

I think, to answer your question or rather solve your problem, the easiest way to do that would be to build the Dockerfile for each of the tools individually and then if you want to run them separately, you could simply do a docker run <tool_name>. That would essentially be the same as running the tool standalone, the only difference being that you would be running that tool inside a docker container now in a much cleaner way. 

For this orchestration, I think the combination of tools that I have right now are fine. If they are not working or if you encounter issues within the orchestration, please let me know. As far as the speed is concerned, I know that's a problem with bigger wordlists. It is definitely something I would have to think about how to go about solving it but right now, that is not a priority. As far as the tools themselves are concerned, unfortunately, I can't do much about it as I am relying on the tool maintainers to make sure they run okay. If the tools run fine, my orchestration would work fine as well.  

Long story short, I don't think I will be able to implement this specific request because I feel this kind of defeats the main purpose of brutesubs
